cmd/signature-v4-utils_test.go

```diff
@@ -243,7 +243,7 @@ func TestGetContentSha256Cksum(t *testing.T) {
 		expected string // expected SHA256
 	}{
 		{"shastring", "", "shastring"},
-		{"", "", emptySHA256},
+		{"", "", ""},
```

Comment:
If the SHA256 of empty string is given also, the request should work is my understanding. So we can keep the test for that.

Reply:
Done